projects/adf4368/Makefile

```diff
@@ -0,0 +1,12 @@
+# Uncomment to use the desired platform
+PLATFORM = mbed
+
+# Select the example you want to enable by chossing y for enabling and n for disabling
+BASIC_EXAMPLE = n
```

Comment:
(commenting something that i commented on another PR that is being reviewed right now)

i don't think editing Makefiles is good to set the example to build even though we have existing projects done this way... let's strive to make the user interaction easy so that they run one of these commands to choose the example to build:

```
make EXAMPLE=basic_example
make EXAMPLE=iio_example
```

Basically you could default to basic like this:
```
EXAMPLE ?= basic_example
```

And take action based on it:
```
ifeq ($(INTERFACE), iio_example)
...
endif
```

Reply:
If its possible I would like to continue with traditional approach. Our very similar product ADF4382 uses the traditional approach. So we want them to be really similar for the customer experience.

Reply:
Fixed